### ISSUE TITLE
Create IF NOT EXISTS throws an error if the stream/table exists

### ISSUE THREAD

**Issue by spena** [confluentinc/ksql#6050]
**Describe the bug**

The `IF NOT EXISTS` clause for `CREATE STREAM|TABLE` does not work as expected. It still throws an error if the stream/table already exists.

The below error is in RED color highlighting the returned error.
```
ksql> create stream if not exists topic1(id int) with (kafka_topic='t1', value_format='json');
Cannot add stream 'TOPIC1': A stream with the same name already exists
```

**To Reproduce**
```
ksql> create stream topic1 (id int) with (kafka_topic='t1', value_format='json', replicas=1, partitions=1);
 Message        
----------------
 Stream created 
----------------

ksql> create stream if not exists topic1 (id int) with (kafka_topic='t1', value_format='json');
Cannot add stream 'TOPIC1': A stream with the same name already exists
```

**Expected behavior**
The `IF NOT EXIST` should not fail if the stream/table already exits.
```
ksql> create stream if not exists topic1 (id int) with (kafka_topic='t1', value_format='json');

  Message
---------------------------------------------------------------------------------------------------
Cannot add stream 'TOPIC1': A stream with the same name already exists
---------------------------------------------------------------------------------------------------
```

It's weird the above message is printed, like `DROP STREAM IF EXISTS <a-stream>`, but seeing the `Message` word means the command did not fail. Seems this is just a warning message.

**Comment by spena** [comment]
This bugfix is being tracked in https://github.com/confluentinc/ksql/pull/6073#discussion_r482038482

**Comment by ashish-padakannaya** [comment]
Nice! Any timeline on when this will be available in an official release?
More specifically, when will it be available in a [docker hub release](https://hub.docker.com/r/confluentinc/cp-ksqldb-server)?

**Comment by spena** [comment]
@ashish-padakannaya this fix is tracked in https://github.com/confluentinc/ksql/pull/6073#discussion_r482038482. Hopefully will be available in the next 0.14 release.
